zrml/prediction-markets/src/benchmarks.rs

```diff
@@ -527,6 +535,145 @@ benchmarks! {
             Ok(())
         })?;
     }: { Pallet::<T>::start_subsidy(&market_clone.unwrap(), market_id)? }
+
+    market_status_manager {
+        let b in 1..31;
+        let f in 1..31;
+        // let t in 0..10;
```

Comment:
if not required please remove.

Reply:
I would like ask, you, @maltekliemann and @sea212, if we like to benchmark the market_status_manager in the case, that the last time frame is far behind the current time frame [here](https://github.com/zeitgeistpm/zeitgeist/blob/30b7191666809c5f40293b0e815c9336d3a9ec91/zrml/prediction-markets/src/lib.rs#L2199-L2209) ?  I think as I benchmarked it currently, it's sufficient enough, because of this:

```rust
market_ids_per_time_frame_len = market_ids_per_time_frame_len
                    .saturating_add(market_ids_per_time_frame.len() as u32);
```

So if we have more than 1 iteration in the parent for-loop, when we simply put a higher benchmark component inside the weight function. 

If you agree, then I would delete the comments, as you suggested.

Reply:
this is interesting point, for my learning could you please explain substrate side issues for bench-marking nested for-loops in bit more detail?

Reply:
Cool, that you ask. I did pick that up in this video [here](https://www.youtube.com/watch?v=Qa6sTyUqgek). I am not quite sure anymore, but I think the benchmarking pallet assumes, that each benchmark component scales linear with higher values. So for example, this is a weight function:

Weight_function = A*x + B*y

Assume x and y are the components. Now the benchmark is basically manipulating x while y is constant and after that manipulating y while x is constant.

But this weight function is not quadratic like this for example: A * x^2 + B * y

Substrates assumes always linear benchmarks.

Reply:
f7f47a5638c99e214ad2166c1d980e96f487bad9